packages/wallets/src/features/cfd/flows/CTrader/AddedCTraderAccountsList/AddedCTraderAccountsList.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useCtraderAccountsList } from '@deriv/api';
+import { TradingAccountCard } from '../../../../../components';
+import { WalletButton, WalletText } from '../../../../../components/Base';
+import CTrader from '../../../../../public/images/ctrader.svg';
+import './AddedCTraderAccountsList.scss';
+
+const AddedCTraderAccountsList: React.FC = () => {
+    const history = useHistory();
+    const { data: cTraderAccounts } = useCtraderAccountsList();
+
+    return (
+        <div className='wallets-added-ctrader-accounts'>
+            <TradingAccountCard
+                leading={() => (
+                    <div className='wallets-added-ctrader-accounts__icon'>
```

Comment:
@thisyahlen-deriv isn't this better we move this rendered component outside and just call them here?

Reply:
By keeping the leading and trailing components inside the TradingAccountCard, we can ensure that they are always rendered together with the card and that they are properly aligned with the card content.

Reply:
@thisyahlen-deriv from outside I mean outside the scope of component but in the same file